docs/generate_class_diagram.sh

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pushd ..
+goplantuml -recursive . > class_diagram.uml
+java -jar ~/tools/plantuml.jar class_diagram.uml
```

Comment:
Do we have in docs how to install this tool?

Reply:
dunno, probably not